### ISSUE TITLE
ui5-textarea: [a11y] aria-required takes no effect

### ISSUE THREAD

**Issue by zhangliangyi** [SAP/ui5-webcomponents#2108]
Dear colleagues,

I am from Successfactors web experience Shanghai team.

**Describe the bug**
No required information output from screen reader when `aira-required` property is set to `true`.

**To reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/Label/
2. Move to section `Label 'for'`
3. Focus on the user input fields marked as required.

**Expected behavior**
Some required information should be output by screen reader.

**Context**
- UI5 Web Components version: 0.23.0
- OS/Platform: macOS 10.15.6
- Browser *(if relevant)*: Microsoft Edge
- Other information: screen reader - NVDA

**Affected components** *(if known)*
- ui5-textarea
- ui5-input
- ui5-select
- ui5-datepicker
- ui5-multi-input

Thanks
Jacob

**Comment by fifoosid** [comment]
The issue has been resolver for all of the mentioned components.
